modules/.htaccess

```
@@ -1,4 +1,4 @@
-<FilesMatch "\.tpl$">
+<FilesMatch "\.tpl|\.twig|config\.xml|config_([a-z]+)\.xml$">
```

If you don't put the `()` only `config_([a-z]+)\.xml` will match the end of line ;)
```suggestion
<FilesMatch "(\.tpl|\.twig|config\.xml|config_([a-z]+)\.xml)$">
```